### DISCUSSION TITLE
Observability/metrics

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#187] [General]
# Observability/metrics

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/Observability/metrics.html

**Comment by chainray01** [comment]
从存储上来讲所有的监控指标都是相同的，但是在不同的场景下这些指标又有一些细微的差异。以 Prometheus 为例，它支持四种不同的指标类型
这里似乎没有写完？

**Reply by isno** [reply]
可观测这一章还没有完稿，metrics、logging 这两篇还需要完善

**Comment by zjyyzhengli** [comment]
“**图 9-17** 展示了一个 Grafana 仪表板（Dashboard）” -- 这个图片编号错了，应该是**图 9-5**

**Reply by isno** [reply]
感谢，已修改。
